native/src/components/NewsListItem.tsx

```diff
@@ -41,11 +43,11 @@ const ListItemWrapper = styled.View`
 const StyledTouchableOpacity = styled.TouchableOpacity`
   flex-direction: column;
 `
-const Divider = styled.View`
+const Divider = styled(View)<{ firstItem?: boolean }>`
```

Comment:
Why did you change `.View` to `(View)`?

Reply:
```suggestion
const Divider = styled(View)<{ firstItem: boolean }>`
```

Reply:
hm firstItem should be optional for the divider!?

Reply:
Sounds good :)